lib/i18n_tools/scanner.rb

```diff
@@ -1,7 +1,7 @@
 module I18nTools
   class Scanner
-    VIEW_REGEXPS = [/[^\w]t\(\"(.*?)\"(.*?)\)/, /[^\w]t\(\'(.*?)\'(.*?)\)/]
-    CODE_REGEXPS = [/I18n\.t\(\"(.*?)\"(.*?)\)/, /I18n\.t\(\'(.*?)\'(.*?)\)/]
+    VIEW_REGEXPS = [/[^\w]t\(\"(.*?)\"(.*?)\)/, /[^\w]t\(\'(.*?)\'(.*?)\)/, /[^\w]t \'(.*?)\'(.*?)/, /[^\w]t \"(.*?)\"(.*?)/]
```

Comment:
That (.*?) at the end will match until the end of the line, which is not what we want here. We want to match until the end of the parameter list, which I don't think is possible if there is no closing parenthesis.